src/main/java/Command.java

```diff
@@ -1,5 +1,8 @@
 import java.io.IOException;
 
+/**
+ * Represents a command to be executed.
+ */
 public abstract class Command {
     String str;
```

Comment:
Could have better names for your variables. I have no idea what str does just from the name. Or maybe you could add a comment to explain what it does...?